integration-tests/ws-client-notification.js

```diff
@@ -200,7 +200,7 @@ describe('WebSocket API Client Notification', function () {
                     requestId: requestId
                 })
                 .assert(function (result) {
-                    utils.hasPropsWithValues(result.notification, ['id', 'notification', 'deviceGuid', 'timestamp']);
+                    utils.hasPropsWithValues(result.notification, ['id', 'timestamp']);
```

Comment:
@zubrabubra according to http://devicehive.com/restful/#WsReference/Client/notificationinsert in response there is notification object that has only two fields: id and timestamp.